visualization.shadertoy/screensaver.shadertoy.txt

```diff
@@ -0,0 +1 @@
+visualization.shadertoy https://github.com/notspiff/visualization.shadertoy master
```

Comment:
well spoted, c&p strikes again fixed in https://github.com/xbmc/repo-binary-addons/pull/15